<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Cheems</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="Inventory_Frame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color : rgb(42,53,66);
	color: white;
	font: 25 11pt &quot;Microsoft JhengHei Light&quot;;
}
QPushButton{
	background-color : rgb(42,53,66);
	color: white;
	font: 25 11pt &quot;Microsoft JhengHei Light&quot;;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>11</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="topMargin">
           <number>13</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Inventario</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>120</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Inventario1">
            <property name="text">
             <string>Inventario 1</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>35</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Inventario2">
            <property name="text">
             <string>Inventario 2</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>35</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Inventario3">
            <property name="text">
             <string>Inventario 3</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">QFrame{
	background-color : rgb(42,53,66);
}
QPushButton{
	background-color : rgb(42,53,66);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <widget class="QPushButton" name="Open_Button">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>../../../../../../../../../../../../../.designer/backup/assets/588a64e0d06f6719692a2d10.png</normalon>
                </iconset>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>886</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="inventory">
            <property name="font">
             <font>
              <family>CMU Bright</family>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>100</horstretch>
                 <verstretch>100</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>700</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="page_1">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLineEdit" name="lineEditT1">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QTableWidget" name="table1">
                      <property name="maximumSize">
                       <size>
                        <width>16777000</width>
                        <height>16777000</height>
                       </size>
                      </property>
                      <column>
                       <property name="text">
                        <string>Nombre</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Categoria</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Cantidad</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Contenedor</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Estado</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame_7">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>1</width>
                        <height>1</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>10000</width>
                        <height>19000</height>
                       </size>
                      </property>
                      <property name="layoutDirection">
                       <enum>Qt::RightToLeft</enum>
                      </property>
                      <property name="autoFillBackground">
                       <bool>false</bool>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton{
	margin: 0px;
	padding:10px;
}</string>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="midLineWidth">
                       <number>1</number>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,12">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="sizeConstraint">
                        <enum>QLayout::SetDefaultConstraint</enum>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QPushButton" name="agregar">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>100000</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>CMU Bright</family>
                          </font>
                         </property>
                         <property name="text">
                          <string>Agregar</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="eliminar">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="font">
                          <font>
                           <family>CMU Bright</family>
                          </font>
                         </property>
                         <property name="text">
                          <string>Eliminar</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="copiar">
                         <property name="font">
                          <font>
                           <family>CMU Bright</family>
                          </font>
                         </property>
                         <property name="text">
                          <string>Copiar</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_2">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLineEdit" name="lineEdit2"/>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_5">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QTableView" name="table2"/>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame_8">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout">
                       <item>
                        <widget class="QPushButton" name="Agregar2">
                         <property name="text">
                          <string>Agregar registro</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="Eliminar2">
                         <property name="text">
                          <string>Eliminar</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="Copiar2">
                         <property name="text">
                          <string>Copiar</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_3">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QFrame" name="frame_6">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Inventario 3</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTableWidget" name="table3">
                      <column>
                       <property name="text">
                        <string>Nombre</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Categoria</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Cantidad</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Contenedor</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Descripcion</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="buttonInv3">
                      <property name="text">
                       <string>PushButton</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
